src/triage/templates/triage/wiki_list.html

```diff
@@ -67,4 +104,18 @@ <h4>Wiki Articles</h4>
             document.location.href = `/wiki/${slug}`;
         }
     });
-{% endblock %}
+
```

Comment:
I think there is an error on the how the jinja is handling the {% endblock %}

Reply:
sorry, mis-read it's with

```
-{% endblock %}
\ No newline at end of file
+{% endblock %}
```

add a newline and it seems like it should be good